flytekit/configuration/internal.py

```diff
@@ -57,6 +57,15 @@ class GCP(object):
     GSUTIL_PARALLELISM = ConfigEntry(LegacyConfigEntry(SECTION, "gsutil_parallelism", bool))
 
 
+class AZURE(object):
+    SECTION = "azure"
+    STORAGE_ACCOUNT_NAME = ConfigEntry(LegacyConfigEntry(SECTION, "storage_account_name"))
```

Comment:
I would have added yaml configs too but I wasn't really sure about how to namespace them given the options that already exist for S3. I feel like there should be separate sections in the yaml config for azure, google and aws but the aws stuff is currently just `storage.connection.endpoint`.

Reply:
we are trying to stay away from configs, as they are not easily portable

Reply:
To clarify, do you mean stay away from the yaml config in favour of the environment variable config, or more generally avoid configs?

Reply:
Hey @Tom-Newton - ketan's just saying that we don't need to make them yaml configs because the yaml format was designed for cross compatibility with `flytectl`'s yaml format.  Unf. though, flytectl will crap out if it sees something in the config file that it doesn't recognize (on the list of things to eventually add).

And we shouldn't add it to flytectl, it doesn't belong there.

One more point to note is that while these are needed to make the azure connection work, these are not great to tote around in a config file.  you should have these set by propeller in the default-env-vars config for the k8s plugin.  If you store these in a text file, they become not portable because in addition to the task definition, you somehow have to carry around this file.

Reply:
Thanks for clarifying :+1:  Personally I thought env vars made more sense for this but the naming of `LegacyConfigEntry` made me think the project preference would have been yaml configs.